sdks/go/pkg/beam/core/runtime/graphx/translate_test.go

```diff
@@ -87,29 +89,38 @@ func TestMarshal(t *testing.T) {
 		name                     string
 		makeGraph                func(t *testing.T, g *graph.Graph)
 		edges, transforms, roots int
+		requirements             []string
 	}{
 		{
 			name: "ParDo",
 			makeGraph: func(t *testing.T, g *graph.Graph) {
-				addDoFn(t, g, pickFn, g.Root(), []*graph.Node{newIntInput(g)}, []*coder.Coder{intCoder(), intCoder()})
+				addDoFn(t, g, pickFn, g.Root(), []*graph.Node{newIntInput(g)}, []*coder.Coder{intCoder(), intCoder()}, nil)
 			},
 			edges:      1,
 			transforms: 1,
 			roots:      1,
 		}, {
 			name: "ScopedParDo",
 			makeGraph: func(t *testing.T, g *graph.Graph) {
-				addDoFn(t, g, pickFn, g.NewScope(g.Root(), "sub"), []*graph.Node{newIntInput(g)}, []*coder.Coder{intCoder(), intCoder()})
+				addDoFn(t, g, pickFn, g.NewScope(g.Root(), "sub"), []*graph.Node{newIntInput(g)}, []*coder.Coder{intCoder(), intCoder()}, nil)
 			},
 			edges:      1,
 			transforms: 2,
 			roots:      1,
+		}, {
+			name: "SplittableParDo",
```

Comment:
While this is testing the Marshal it's not validating that the Requirements are populated as expected. Consider checking the new change in the resulting proto, if not the length like the other tests.